scenarios/prod/strace/guide_content.yml

```diff
@@ -240,7 +240,7 @@ contentDefinitions:
     content: What is the secret Flag obtained by tracing script.sh? (The inner contents of FLAG{...})
     answers:
       - answer_type: String
-        value: $RANDOM_ONE
+        value: $RANDOM
```

Comment:
fix for strace install script failing because no arguments were being passed to it.